src/main/java/uk/gov/companieshouse/api/accounts/controller/CurrentPeriodController.java

```diff
@@ -111,16 +108,11 @@ public ResponseEntity get(HttpServletRequest request) {
 
         Transaction transaction = (Transaction) request
             .getAttribute(AttributeName.TRANSACTION.getValue());
-        CompanyAccountEntity companyAccountEntity = (CompanyAccountEntity) request
-            .getAttribute(AttributeName.COMPANY_ACCOUNT.getValue());
 
-        if (companyAccountEntity == null) {
-            LOGGER.error("Current Period error: No company account in request session",
-                logContext);
-            return ResponseEntity.status(HttpStatus.NOT_FOUND).body(null);
-        }
+        Map<String, String> pathVariables = (Map) request
```

Comment:
same as above - if we can use a variable passed into the controller method that'd be nicer than pathvariables